mediathread/templates/main/collection_add.html

```diff
@@ -33,40 +35,89 @@
-                        </p>
-                        <h3>Video</h3>
+                    {% if can_upload and uploader %}
+                        <div class="col-md-6 order-md-2">
 
+                            <h2>Upload Media</h2>
+                                <p class="lead">
+                                    Contribute video and audio to this course directly from your desktop into Mediathread with Wardenclyffe.
+                                </p>
 
 
-                        <h3>Audio</h3>
+                                {% flag "uploads_offline" %}
+                                    <div class="instructions">
+                                        <p>The upload feature is currently down for maintenance. We will return shortly.</p>
+                                    </div>
+                                {% else %}
+                                    {% ifequal role_in_course "non-member" %}
+                                        <div class="instructions">
+                                            <p>You must be a course member to upload media files.</p>
+                                        </div>
+                                    {% else %}
+                                        <h3>Video</h3>
+                                        <form action="/upload/redirect/{{uploader.id}}/" method="POST">
 
+                                                <div class="form-row align-items-center">
+                                                    {% if owners %}
+                                                        <div class="col-auto my-1">
+                                                            <label class="mr-sm-2 sr-only" for="video_owners"></label>
+                                                            <select class="custom-select mr-sm-2" id="video_owners" name="as">
+                                                            {% for owner in owners %}
+                                                                <option value="{{owner.username}}" {% ifequal owner.username user.username %} selected="selected"{% endifequal %}>{{owner.public_name}}</option>
+                                                            {% endfor %}
+                                                            </select>
+                                                        </div>
+                                                    {% endif %}
+                                                    <div classs="col-auto my-1">
+                                                        <button type="submit" class="btn btn-secondary upload_button video">Upload Video</button>
+                                                    </div>
+                                                </div>
+                                        </form>
+                                        <h3>Audio</h3>
```

Comment:
Add a little spacing here - maybe `mt-1 or 2`

Reply:
Got it!